edx/analytics/tasks/insights/enrollments.py

```diff
@@ -77,6 +77,12 @@ def mapper(self, line):
             self.incr_counter(self.counter_category_name, 'Discard Non-Enrollment Event Type', 1)
             return
 
+        username = eventlog.get_event_username(event)
```

Comment:
Do we have access to the email in this context?

Ideally, I'd think we'd use the same filtering method (invalid domain) as in platform.

Reply:
No we don't. We have to work with what the event log provides.

Reply:
Bummer, +1